content/applications/finance/expenses.rst

```diff
@@ -1,12 +1,13 @@
 :show-content:
+:show-toc:
```

Comment:
OK- I believe I have this also in my main Recruitment doc, so I copied it form there. I'll be sure to NOT include these when I have a seealso form now on! (it might be in another final review doc already though!)